docs/layouts/partials/scripts/user-customizations.html

```diff
@@ -3,9 +3,11 @@
 <script src="{{"scripts/user-widget.js" | relURL}}"></script>
 
 <!-- Google Tag Manager -->
+{{- if gt ( len ( getenv "PLATFORM_PROJECT" ) ) 0 -}}
```

Comment:
```suggestion
{{- if and ( gt ( len ( getenv "PLATFORM_PROJECT" ) ) 0 ) ( ne "master"  ( getenv "PLATFORM_BRANCH" )) -}}
```

Reply:
This ain't gonna work, since `PLATFORM_BRANCH` isn't available during builds....